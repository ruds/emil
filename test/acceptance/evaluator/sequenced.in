(12; "foo");
(12; "foo") : string;
