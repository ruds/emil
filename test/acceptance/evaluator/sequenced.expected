val it = "foo" : string
val it = "foo" : string
